src/components/ExpComponent.jsx

```diff
@@ -1,184 +1,136 @@
-                                <HiFilter /> Filter
-                            </button>
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div id="tabs" className="dropdown is-hidden-desktop">
-                <div className="dropdown-trigger">
-                    <button className="button is-rounded is-primary is-light ml-3" style={{ width: "20vw" }} aria-haspopup="true" aria-controls="dropdown-menu" onClick={DropTabs}>
-                        <span>Select Tab</span>
-                        <span className="icon is-small">
-                            <i className="fas fa-angle-down" aria-hidden="true"></i>
-                        </span>
-                    </button>
-                </div>
-                <div className="dropdown-menu" id="dropdown-menu" role="menu" style={{ width: window.screen.width }}>
-                    <div className="dropdown-content">
-                        <a id="pop-tab" href="#" className="dropdown-item is-active" onClick={LoadPop}>
-                            Popular
-                        </a>
-                        <a id="rec-tab" className="dropdown-item" onClick={LoadRecents}>
-                            Recents
-                        </a>
-                        <a id="all-tabs" href="#" className="dropdown-item" onClick={LoadAll}>
-                            All Experiments
-                        </a>
-                        <a id="save-tabs" href="#" className="dropdown-item" onClick={LoadSaved}>
-                            Saved
-                        </a>
-                    </div>
-                </div>
+const NextPage = () => {
+  setPage(page + 1);
+};
+
+React.useEffect(() => {
+  fetch("https://8kne7udek3.execute-api.ap-southeast-2.amazonaws.com/items")
+    .then((resp) => resp.json())
+    .then((data) => {
+      console.log(data);
+      setExperiments(data);
+      setTotal(Math.ceil(data.length / 8));
+      let copy = [...data];
+      setPop(copy.sort((a, b) => b["Pageviews"] - a["Pageviews"]));
+      setFetched(1);
+    });
+}, []);
+
+const SearchExp = (e) => {
+  setWord(e.target.value);
+};
+
+const ToggleFilter = () => {
+  setFilterActive(!filterActive);
+  console.log(document.getElementById("filter-set").className);
+  if (!document.getElementById("filter-set").className.includes("is-hidden-desktop is-hidden-tablet")) {
+    document.getElementById("filter-set").className = document.getElementById("filter-set").className + " is-hidden-desktop is-hidden-tablet";
+  } else {
+    document.getElementById("filter-set").className = 'column is-2 is-hidden-mobile is-hidden-tablet-only mr-4';
+  }
+  document.getElementById("filter-model").className = document.getElementById("filter-model").className + " is-active";
```

Comment:
You can use add method of classList property for better readability. Look [here](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)